kinto/core/storage/testing.py

```diff
@@ -1238,6 +1238,22 @@ def test_parent_cannot_update_other_parent_record(self):
                                        **self.storage_kw)
         self.assertNotIn("another", not_updated)
 
+    def test_bytes_value_gets_back_str(self):
+        data = {'steak': 'haché'.encode(encoding='utf-8')}
+        self.assertIsInstance(data['steak'], bytes)
+
+        record = self.create_record(data)
+
+        back_record = self.storage.get(object_id=record['id'],
+                                       **self.storage_kw)
+        self.assertIsInstance(back_record['steak'], str)
+
+    def test_bytes_value_bad_encoding_raises(self):
+        self.assertRaises(Exception,
```

Comment:
Can we `self.assertRaises((OverflowError, UnicodeDecodeError), ...)`? Per https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertRaises, it seems like we can.